packages/test/local-server-tests/src/test/opsOnReconnect.spec.ts

```diff
@@ -109,22 +108,20 @@ describe("Ops on Reconnect", () => {
             const message = unpackRuntimeMessage(containerMessage);
             if (message.type === ContainerMessageType.FluidDataStoreOp) {
                 const envelope = message.contents as IEnvelope;
-                if (envelope.address !== agentSchedulerId) {
```

Comment:
I do not know why agentSchedulerId was excluded here. Are we about to break something? :)

Reply:
It was just filtering out the agent scheduler ops (you can see below that it assumes the contents of the op are for a `SharedMap` or `SharedDirectory`).  Since this test is using `ContainerRuntime` _without_ `AgentScheduler` after this change, the filtering is no longer necessary.